main.go

```diff
@@ -12,6 +13,7 @@ import (
 	"github.com/fyne-io/fyne/app"
 	"github.com/fyne-io/fyne/layout"
 	"github.com/fyne-io/fyne/widget"
+	xkcd "github.com/steveoc64/examples/xkcd"
```

Comment:
For the merge to succeed this would have to be "fyne-io/examples/xkcd", also the import name is not needed.

```suggestion
	"github.com/fyne-io/examples/xkcd"
```